rs_gazebo/worlds/stairs_only.world

```diff
@@ -0,0 +1,297 @@
+<?xml version="1.0"?>
+<sdf version='1.6'>
+  <world name='default'>
+    <light name='sun' type='directional'>
+      <cast_shadows>0</cast_shadows>
+      <pose frame=''>0 0 10 0 -0 0</pose>
+      <!--<diffuse>0.8 0.8 0.8 1</diffuse>-->
+      <!--<specular>0.2 0.2 0.2 1</specular>-->
+      <!--<attenuation>-->
+        <!--<range>1000</range>-->
+        <!--<constant>0.9</constant>-->
+        <!--<linear>0.01</linear>-->
+        <!--<quadratic>0.001</quadratic>-->
+      <!--</attenuation>-->
+      <!--<direction>-0.5 0.1 -0.9</direction>-->
+    </light>
+    <model name='ground_plane'>gazebo.launch
```

Comment:
Please check this line. looks like it has errors

Reply:
It is typo, thanks.